src/content/guides/creating-clusters-via-crs-on-aws/index.md

```diff
@@ -0,0 +1,169 @@
+  - [AWSCluster](/reference/cp-k8s-api/awsclusters.infrastructure.giantswarm.io/) - represents a tenant cluster's Kubernetes control plane.
+  - [AWSMachineDeployment](/reference/cp-k8s-api/awsmachinedeployments.infrastructure.giantswarm.io/) -  represents tenant cluster node pools.
+
+
+## Cluster CRs
+
+*Cluster* CR object example:
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1alpha2
+kind: Cluster
+metadata:
+  generation: 1
+  labels:
+    cluster-operator.giantswarm.io/version: 2.1.1
+    giantswarm.io/cluster: nzr5z
+    giantswarm.io/organization: giantswarm
+    release.giantswarm.io/version: 11.0.1
+  name: nzr5z
+  namespace: default
+spec:
+  infrastructureRef:
+    apiVersion: infrastructure.giantswarm.io/v1alpha2
+    kind: AWSCluster
+    name: nzr5z
+    namespace: default
+```
+
+*AWSCluster* CR object example:
+
+```yaml
+apiVersion: infrastructure.giantswarm.io/v1alpha2
+kind: AWSMachineDeployment
+metadata:
+  labels:
+    aws-operator.giantswarm.io/version: 8.1.1
+    giantswarm.io/cluster: nzr5z
+    giantswarm.io/machine-deployment: pv7ps
+    giantswarm.io/organization: giantswarm
+    release.giantswarm.io/version: 11.0.1
+  name: pv7ps
+  namespace: default
+spec:
+  nodePool:
+    description: np1
+    machine:
+      dockerVolumeSizeGB: 100
+      kubeletVolumeSizeGB: 100
+    scaling:
+      max: 10
+      min: 3
+  provider:
+    availabilityZones:
+    - eu-central-1a
+    worker:
+      instanceType: m4.xlarge
+```
+
+## MachineDeployment CRs
+
+*MachineDeployment* CR object example:
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1alpha2
+kind: MachineDeployment
+metadata:
+  labels:
+    cluster-operator.giantswarm.io/version: 2.1.1
+    giantswarm.io/cluster: nzr5z
+    giantswarm.io/machine-deployment: pv7ps
+    giantswarm.io/organization: giantswarm
+    release.giantswarm.io/version: 11.0.1
+  name: pv7ps
+  namespace: default
+spec:
+  selector: {}
+  template:
+    metadata: {}
+    spec:
+      bootstrap: {}
+      infrastructureRef:
+        apiVersion: infrastructure.giantswarm.io/v1alpha2
+        kind: AWSMachineDeployment
+        name: pv7ps
+        namespace: default
+      metadata: {}
+```
+
+*AWSMachineDeployment* CR object example:
+
+```yaml
+apiVersion: infrastructure.giantswarm.io/v1alpha2
+kind: AWSMachineDeployment
+metadata:
+  labels:
+    aws-operator.giantswarm.io/version: 8.1.1
+    giantswarm.io/cluster: nzr5z
+    giantswarm.io/machine-deployment: pv7ps
+    giantswarm.io/organization: giantswarm
+    release.giantswarm.io/version: 11.0.1
+  name: pv7ps
+  namespace: default
+spec:
+  nodePool:
+    description: np1
+    machine:
+      dockerVolumeSizeGB: 100
+      kubeletVolumeSizeGB: 100
+    scaling:
+      max: 10
+      min: 3
+  provider:
+    availabilityZones:
+    - eu-central-1a
+    worker:
+      instanceType: m4.xlarge
+```
+
+## How to create the cluster omitting Giant Swarm API?
```

Comment:
'omitting' feels weird - without? 

alternatively, it's possibly better to have this in the positive, e.g: 'How to create a cluster using Cluster API?', if you see what i mean

also, 'a' cluster, not 'the' cluster